charts/nextcloud/values.yaml

```diff
@@ -355,6 +355,15 @@ redis:
     existingSecret: ""
     # Password key to be retrieved from existing secret
     existingSecretPasswordKey: ""
+  # Since Redis is used for caching only, you might want to use a storageClass with different reclaim policy and backup settings
+  global:
+    storageClass: ""
+  master:
+    persistence:
+      enabled: true
+  replica:
+    persistence:
+      enabled: true
```

Comment:
I agree that persistence should be off by default. But in the current chart it's enabled by default. So if we disable it now, it might lead to unpleasant surprises for people who are relying on the default values.

So given that it's currently enabled by default, I would suggest to keep it that way.